test/e2e/functional_test.go

```diff
@@ -28,7 +28,7 @@ func (s *FunctionalSuite) TestArchiveStrategies() {
 		Workflow(`@testdata/archive-strategies.yaml`).
 		When().
 		SubmitWorkflow().
-		WaitForWorkflow(time.Minute).
+		WaitForWorkflow(time.Minute + time.Second*30).
```

Comment:
nitpick: could we change these to `time.Second*90` for more conciseness and one less computation?

Reply:
Yeah, good point.